### ISSUE TITLE
Zero-fee transaction

### ISSUE THREAD

**Issue by thanhson1085** [BuildOnViction/tomochain-v1#123]
I make a zero-fee tx, it works. But It should require at least 1 wei transaction fee.

```
> eth.getTransaction('0x7b6cb25e5143ac5e75ad20260a57b54da42701f0600856e24b739b99f4e88ab5')
{
  blockHash: "0x8b4755c8068f2ef4357a3452efe4305b60fd13219b6eb23c0a096d26ed371e1f",
  blockNumber: 1901,
  from: "0x487d62d33467c4842c5e54eb370837e4e88bba0f",
  gas: 125326,
  gasPrice: 0,
  hash: "0x7b6cb25e5143ac5e75ad20260a57b54da42701f0600856e24b739b99f4e88ab5",
  input: "0x02aa9be200000000000000000000000031b249fe6f267aa2396eb2dc36e9c79351d97ec50000000000000000000000000000000000000000000000000de0b6b3a7640000",
  nonce: 11,
  r: "0x5a7223d6c9628b28d7e635f21df9e41526f24b5c5fec3b1b79f07934d0f9c79",
  s: "0x23900deb3975138421b7a6f050adf7bc582fb5e5a283961662788abd57b185e0",
  to: "0x0000000000000000000000000000000000000088",
  transactionIndex: 2,
  v: "0xd5",
  value: 0
}
```

**Comment by thanhson1085** [comment]
After investigation, this is not a bug. For default, gasPrice 0 tx will be process, unless it is remove from pool at checkpoint

**Comment by thanhson1085** [comment]
fixed #128
